database/migrations/create_forms_table.php

```diff
@@ -4,8 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Support\Facades\Schema;
 
-return new class extends Migration
-{
+return new class () extends Migration {
```

Comment:
personally i would have set the primary key of `form_id` just to `->id()`
it's a minor thing